policy/policy-01-policy-enforcement/policy-enforcement-consumer/build.gradle.kts

```
@@ -1,3 +1,17 @@
+/*
+ *  Copyright (c) 2023 Fraunhofer Institute for Software and Systems Engineering
```

nit: 
```suggestion
 *  Copyright (c) 2024 Fraunhofer Institute for Software and Systems Engineering
```

same for other headers